WordPress/WordPressUITests/Flows/EditorFlow.swift

```diff
@@ -1,3 +1,4 @@
+import UITestsFoundation
```

Comment:
You'll see a few of these import statements in files from the other UI tests targets. The reason they are here is because these tests used one or more of the utils that have been extracted into the framework.